test/Microsoft.Azure.Devices.Edge.Test.Common/certs/RootCertificateAuthority.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Certs
+{
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    public class RootCertificateAuthority
+    {
+        readonly string scriptPath;
+
+        public static async Task<RootCertificateAuthority> CreateAsync(string certificatePath, string keyPath, string password, string scriptPath, CancellationToken token)
```

Comment:
it looks weird that constructor is not at the top? did you format this file using resharper?

Reply:
Yes Resharper doesn't complain--I didn't think Resharper cared about that kind of thing by default. I'm happy to put the ctor first if that's the style. Where should static functions (esp factory functions) go?